platform/vs/docker-ptf.mk

```diff
@@ -2,6 +2,6 @@
 
 DOCKER_PTF = docker-ptf.gz
 $(DOCKER_PTF)_PATH = $(DOCKERS_PATH)/docker-ptf
-$(DOCKER_PTF)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF) $(PYTHON_SAITHRIFT)
+$(DOCKER_PTF)_DEPENDS += $(LIBTHRIFT) $(PYTHON_THRIFT) $(PTF) $(PYTHON_SAITHRIFT) $(SWSS) $(LIBTEAMDCT) $(LIBTEAM_UTILS)
```

Comment:
>LIBTEAMDCT) $(LIBTEAM_UTILS [](start = 92, length = 27)

Could you use debian vanilla packages?
No need to install their dependencies explicitly.

Reply:
I have confirmed that the teamd from vanilla package can be used.